docs/server/security.md

```diff
@@ -73,7 +73,7 @@ The following arguments must be passed to the `Authorize` method for example:
 <!--SNIPSTART temporal-common-authorization-authorizer-calltarget-->
 <!--SNIPEND-->
 
-The `Authorize` method then returns one of two possible decisions within the `Result.Decision` field:
```

Comment:
This is another decision. Please revert.

Reply:
good catch